content/en/developers/metrics/distributions.md

```diff
@@ -44,9 +42,10 @@ Percentile aggregations can be added in-app at the [Datadog Distribution Metric
 
 ### Customization of tagging
 
-This functionality allows you to control tagging for metrics for which host-level granularity is not necessary see the [Distribution Metric page][3] to learn more.
+This functionality allows you to control tagging for metrics for which host-level granularity is not necessary see the [Distribution Metric page][3] to learn more about our whitelist-based tagging control. Note: the exclusion of tags with `!` will not be accepted in this customization feature.
 
 ## Submission
+Note: As distributions are a new metric type, you should submit distributions under new metric names.
```

Comment:
```suggestion
Since distributions are a different metric type, any previously submitted metrics should be instrumented under new metric names.
```

Reply:
Not necessarily that any previously submitted metrics should be instrumented under new metric names...just that distributions shouldn't have the same name as previously submitted metric types. 

Might be easier to say Note: Different metric types in Datadog should be submitted under different metric names

Reply:
updated